src/widgets/DataTable/TouchableNoFeedback.js

```diff
@@ -17,7 +17,7 @@ import PropTypes from 'prop-types';
 const onPressNoOp = () => {};
 
 const TouchableNoFeedback = ({ children, style, ...touchableProps }) => (
-  <TouchableWithoutFeedback onPress={onPressNoOp} {...touchableProps}>
+  <TouchableWithoutFeedback {...touchableProps} onPress={onPressNoOp}>
```

Comment:
What does this fix involve? Were the unpacked parameters overriding `onPress`? If so, might be worth making the intended override explicit, maybe something like:

```
const TouchableNoFeedback = ({ children, style, ...touchableProps }) => {
  const touchableWithoutFeedbackProps = { ...touchableProps, onPress: onPressNoOp };
  return <TouchableWithoutFeedback {...touchableWithoutFeedbackProps}>;
}
```

Reply:
yeah, overriding.

I think your suggestion will cause a new object and unneccessary re-renders, I can try it out

Reply:
No worries. Will merge.